instagram/graminsta/src/App.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import Searchbar from "./Components/Searchbar.js"
+import PostContainer from "./Components/PostContainer.js"
+import CommentSection from "./Components/CommentSection.js"
+import './App.css';
+
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <Searchbar />
+        <PostContainer />
+        <CommentSection />
+      </div>
```

Comment:
your comments might end up needing to go inside you posts to easily tie them to the individual post as each post will have its own comment section